code/modules/events/brain_trauma.dm

```diff
@@ -0,0 +1,34 @@
+/datum/round_event_control/brain_trauma
+	name = "Spontaneous Brain Trauma"
+	typepath = /datum/round_event/brain_trauma
+	weight = 25
+
+/datum/round_event/brain_trauma
+	fakeable = FALSE
+
+/datum/round_event/brain_trauma/start()
+	for(var/mob/living/carbon/human/H in shuffle(GLOB.alive_mob_list))
+		if(!H.client)
+			continue
+		if(H.stat == DEAD) // What are you doing in this list
+			continue
+		if(!H.getorgan(/obj/item/organ/brain)) // If only I had a brain
+			continue
+
+		traumatize(H)
+		announce_to_ghosts(H)
+		break
+
+/datum/round_event/brain_trauma/proc/traumatize(mob/living/carbon/human/H)
+	var/resistance = pickweight(list(
+		TRAUMA_RESILIENCE_BASIC = 50,
+		TRAUMA_RESILIENCE_SURGERY = 30,
+		TRAUMA_RESILIENCE_LOBOTOMY = 15,
+		TRAUAM_RESILIENCE_MAGIC = 5))
```

Comment:
I think it's probably better to limit it to lobotomy, just so there's always something you can do about it

Reply:
Doesn't cloning remove all brain traumas anyway

Reply:
it's funnier if you get magic, but trauma is spelt wrong on it?

Reply:
oh yeah that's TRAUAM how is it passing the travis check